backend/src/main/java/com/eyetrade/backend/model/entity/CurrencyFollowing.java

```diff
@@ -35,6 +35,6 @@
     private CurrencyType baseCurrencyType;
 
     @NotNull
-    @Column(name = "timestamp")
-    private Long timestamp;
+    @Column(name = "Date")
+    private String followingDate;
```

Comment:
I suggest you to delete the currency following and currency record tables to avoid future bugs, if you have not. Because Hibernate does not delete the old versions of the fields.

Reply:
I have already deleted, and tested :)

Reply:
Cool :)